kubernetes-deployment/cluster-config-example.yaml

```diff
@@ -184,3 +184,5 @@ workermachinelist:
 
 
 
+
+
```

Comment:
no change in this yaml, excluded from this PR?

Reply:
Resolved.